import { BaseConflictException } from '../../shared/exceptions/conflict'

export class UserAlreadyExistsException extends BaseConflictException {
  constructor(ci: string, error?: Error) {
    super(`Usuario con c√©dula: ${ci} ya existe`, error)
  }
}
